ansible/roles/ocp4-workload-security-compliance-lab/tasks/post_workload.yml

```diff
@@ -1,24 +1,23 @@
 ---
-- name: post_workload Tasks Started
-  debug:
-    msg: "post_workload Tasks - Started"
-
-#####
-#
+# Implement your Post Workload deployment tasks here
 
-- name: Delete the remote files used in this role
+- name: delete the remote files used in this role
   file:
-    path: "{{tmp_dir}}"
+    path: "{{ tmp_dir }}"
     state: absent
   tags: workload
 
-#
-#####
-
-- name: Print email text
+- name: output workshop info
   debug:
-          msg: "user.info: Please run the instructor setup steps first, located here: https://github.com/RedHatDemos/SecurityDemos/blob/master/2019Labs/ProactiveSecurityCompliance/documentation/lab7-instructor-prep.adoc  Next, follow the lab directions located here: https://github.com/RedHatDemos/SecurityDemos/blob/master/2019Labs/ProactiveSecurityCompliance/documentation/lab7.adoc  Use a USERID of user1 when following the lab directions. OpenShift console: https://master.{GUID}.summit.opentlc.com  Username: user1 or admin, password: r3dh4t1!" 
- 
-- name: post_workload Tasks Complete
+    msg:
+    - "Follow the lab directions located here: https://github.com/RedHatDemos/SecurityDemos/blob/master/2019Labs/ProactiveSecurityCompliance/documentation/lab7.adoc"
```

Comment:
Look at https://github.com/redhat-cop/agnosticd/blob/development/ansible/configs/ocp4-disconnected-osp-lab/post_software.yml#L10-L39 for how to do this.